src/app/shared/service/state-service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StateService {
+  getState(stateKey: string): any {
```

Comment:
Missing type, 
`any` just disabled type checking better add type or in cases it's impossible use `unknown`,
but here we can create an interface